cognite/client/utils/_auxiliary.py

```diff
@@ -55,6 +55,18 @@ def get_accepted_params(cls: type[T_CogniteResource]) -> dict[str, str]:
     return {to_camel_case(k): k for k in vars(cls()) if not k.startswith("_")}
 
 
+def load_dict_or_str(resource: dict | str) -> dict:
```

Comment:
Since this method enforces dict, I'd love for the name to reflect that. Maybe load_resource_to_dict? 🤔 

Also, keys must be str, right?
```suggestion
def load_dict_or_str(resource: dict[str, Any] | str) -> dict[str, Any]:
```